CHANGELOG.md

```diff
@@ -21,9 +21,16 @@ x, y, z = ts.get_particle(['x', 'y', 'z'], iteration=1000)
 compatibility for 3d field:
 ```get_field(field=field, coord=coord, iteration=iteration)```
 used to return the central slice along `y` while it now returns the full 3d field.
```

Comment:
I forgot to add the lines below in a previous PR ; sorry to include it here as part of this PR.